### ISSUE TITLE
cert issue in final locations lab (both tracks impacted)

### ISSUE THREAD

**Issue by jarcher** [RedHatWorkshops/dayinthelife-integration#326]
Both labs where the final web application displays the locations on the map is currently broken due what seems to be a certificate issue.  In my instance this webpage 

http://www-user2.apps.houston-e047.openshiftworkshop.com/locations.html

is only detectable by reviewing the dev tools console in the browser to see the cert exception needed to be accepted.

**Comment by VinayBhalerao** [comment]
@jarcher Yep, it's a known issue. It's been highlighted as part of the troubleshooting step https://github.com/RedHatWorkshops/dayinthelife-integration/blob/master/docs/labs/citizen-integrator-track/lab06/walkthrough.adoc#troubleshooting-the-locations-page
